docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,117 @@ Diffraction Changes
-- :ref:`LoadWANDSCD <algm-LoadWANDSCD-v1>` now has a new option to perform normalization in the same loading process.
+  on a scatter plot, new option for self-attenuation treatment using measured transmission.
+- Several improvements have been made to the group calibration routine including
+
+
+	- More input control parameters, including peak function type for estimating offset after cross correlation and an option to turn on or off the smoothing of data for cross correlation purpose.
+	- The workflow of group calibration script is also polished to make it smoother. Accordingly, unit tests have been updated.
+	- Making it more generic.
+	- Groups are now allowed with dedicated control parameters.
+	- Documentation has been added as a guidance for general users.
+
+- :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` major interface update along with enabling the calibration of T0 and sample position.
+- :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` minor interface update that allows fine control of bank rotation calibration.
+- :ref:`SNAPReduce <algm-SNAPReduce-v1>` permits saving selected property names and values to file, to aid autoreduction.
+- Add a custom ttmode to the PEARL powder diffraction scripts for running with a custom grouping file.
+- Added a 3mf format file describing the PEARL sample and environment shapes for the P-E press. Also fixed a couple of minor issues in the 3mf file format loader used in :ref:`LoadSampleEnvironment  <algm-LoadSampleEnvironment>`.
 
 Bugfixes
 ########
 - Fix the issue with the calibration diagnostics script when dealing with instruments of which the detector ID does not start from 0.
 - Fix the issue with :ref:`SNSPowderReduction <algm-SNSPowderReduction>` - when invalid height unit is encountered while reading sample log, we should continue by ignoring geometry and rely purely on user input.
-- fix d-spacing calculation when parabolic model is selected.
+- Fix d-spacing calculation when parabolic model is selected.
 - Correct equation for pseudo-voigt FWHM and mixing parameter in peak profile function :ref:`Bk2BkExpConvPV <func-Bk2BkExpConvPV>`.
 
 Engineering Diffraction
 -----------------------
 New features
 ############
-- New setting for default peak function to fit in the Engineering Diffraction interface (initial default is :ref:`BackToBackExponential <func-BackToBackExponential>`).
-- Added serial fit capability to fitting tab in EngDiff UI - this fits all loaded workspaces with same initial parameters.
+- New setting for default peak function to fit in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` (initial default is :ref:`BackToBackExponential <func-BackToBackExponential>`).
+- Added serial fit capability to :ref:`Fitting tab <ui engineering fitting>` in :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` - this fits all loaded workspaces with same initial parameters.
```

Comment:
nice addition of references to individual tabs in the UI docs (though I haven't built to check they work yet)!

Reply:
This will definitely need checking at some point as the Fitting Tab and Focus Tab links are brand new.